goalgorilla/open_social
# Issue #2840943 by ribel, maikelkoopman: show new comments on nodes

Show new comments since last view on nodes
How to test:
- Enable history module
- Place a new comment on a node
- 'New' label should appear
